frontend/src/components/StateCountySelector/StateCountySelector.tsx

```diff
@@ -38,9 +40,9 @@ let CountySelector: React.StatelessComponent<CountySelectorProps> = ({ onChange,
         hintText='Select a county'
         floatingLabelFixed={true}
         autoWidth={false}
-        style={{width: 150, paddingLeft: 10}}
+        style={{ width: 150, paddingLeft: 10 }}
     >
-        <MenuItem key='Custom' value={true} primaryText='Custom' />
+        <MenuItem key='Custom' value={true} primaryText='Custom' disabled={(ENV !== 'PRD')} />
```

Comment:
should we just get rid of the menu and default to all for prod? As it is with this change, the user will have to select ALL, but they won't actually be able to select anything else. Seems weird.

Reply:
@viviandien, thoughts? For now I disabled the button instead of removing it. This way people need to actively select "All". My idea was to show that custom could be possible, and also avoid loading a state RP without any "confirmation step". @akegan

Reply:
I think it's better UX to have just remove the select ALL step, since it's the only option. I have suggested updates to the copy to make this clear.  The Filter by county name/type in the select all is pretty prominent, so I don't think alluding to a potential custom option is that valuable at this stage.